lib/build/web_components.dart

```diff
@@ -25,7 +25,7 @@ Asset generateWebComponentsBootstrap(Resolver resolver, Transform transform,
     dom.Document document, AssetId scriptId, AssetId newScriptId,
     {List<InitializerPlugin> extraPlugins: const []}) {
   var htmlImportRecorder = new HtmlImportAnnotationRecorder();
-  var plugins = [htmlImportRecorder]..addAll(extraPlugins);
+  var plugins = [htmlImportRecorder as InitializerPlugin]..addAll(extraPlugins);
```

Comment:
k worked. Actually I thought to have tried it before but probably I made some typos.